lms/djangoapps/grades/tests/test_transformer.py

```diff
@@ -100,7 +100,7 @@ def assert_collected_transformer_block_fields(self, block_structure, usage_key,
         for field in expectations:
             self.assertEqual(
                 expectations[field],
-                block_structure.get_transformer_block_field(usage_key, transformer_class, field),
+                block_structure.get_transformer_block_field(usage_key, transformer_class, field).decode('utf-8'),
```

Comment:
Should this decode be happening in the underlying function so that the data we get back is in the same format as the expected data?  Or should the expectation values be updated to be unicode instead?

Reply:
@nasthagiri do you remember if the output of block structures should be unicode or byte streams?

Reply:
@feanil any update ?

Reply:
@awais786 @feanil When looking into this issue, it seems this is already fixed in the latest master branch.  So I believe we can close this PR.

Now, there are only 2 remaining issues in the `grades` app:
* [test_recalculate_grades](https://github.com/edx/edx-platform/blob/50cd2ec409522bed92870284643acd664b555dc2/lms/djangoapps/grades/management/commands/tests/test_recalculate_learner_grades.py#L50), which I'm fixing in https://github.com/edx/edx-platform/pull/21836.
* [test_score_submission_for_all_problems](https://github.com/edx/edx-platform/blob/50cd2ec409522bed92870284643acd664b555dc2/lms/djangoapps/grades/tests/integration/test_problems.py#L76), which is dependent on merging https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/227

Reply:
Great.